nifi-commons/nifi-record/src/test/java/org/apache/nifi/serialization/record/TestDataTypeUtils.java

```diff
@@ -923,23 +939,73 @@ public void testConvertDateToUTC() {
         assertEquals(0, zdt.getNano());
     }
 
+    /**
+     * Convert String to java.sql.Date using implicit default DateFormat with GMT Time Zone
+     *
+     * Running this method on a system with a time zone other than GMT should return the same year-month-day
+     */
     @Test
-    public void testConvertDateToLocalTZ() {
-        int year = 2021;
-        int month = 1;
-        int dayOfMonth = 25;
+    public void testConvertTypeStringToDateDefaultTimeZoneFormat() {
```

Comment:
How do you feel about setting the the user.timezone system property in some of these tests to show it working regardless of the user's time zone?  You'd have to reset the property after the test, of course.  The only thing I'd be concerned about is a multi-threaded build, but I wanted to see what you thought.

Reply:
I thought about that approach, and there is one existing unit test method that works as you described. However, since the GitHub automated builds run with three different time zone settings, that seems to be a better way to cover a variety of system configurations.

Reply:
Good to know, that makes me feel better.